jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/StatisticsHandler.java

```diff
@@ -61,45 +56,32 @@ public boolean process(Request request, Response response, Callback callback) th
         StatisticsRequest statisticsRequest = newStatisticsRequest(request);
         try
         {
-            _processStats.increment();
-            _requestStats.increment();
-
-            String id = statisticsRequest.getConnectionMetaData().getId();
-            if (_connectionStats.add(id))
-                statisticsRequest.getConnectionMetaData().getConnection().addEventListener(statisticsRequest);
-
             if (next.process(statisticsRequest, response, callback))
                 return true;
-            _requestStats.decrement();
+            statisticsRequest.unProcessed();
```

Comment:
Rename to `notProcessed()`? Reads "statistics request not processed".

Reply:
Done.